### DISCUSSION TITLE
Generate report.json in Cucumber JSON output format

### DISCUSSION THREAD

**Discussion by malindajudo** [hrcorval/behavex#113] [General]
Please I want to know whether Is it possible to generate the report.json in Cucumber JSON output format ? I need this to upload details to JIRA Xray test executions via multipart API (https://docs.getxray.app/display/XRAYCLOUD/Import+Execution+Results+-+REST+v2#ImportExecutionResultsRESTv2-CucumberJSONresultsMultipart) and it's expecting the report.json in cucumber json format. 
In behave we can give the format as below but I don't see that option in behavex 
`behave -f cucumber_json:PrettyCucumberJSONFormatter -o cucumber_report.json`

**Comment by hrcorval** [comment]
Hi @malindajudo, 
Last week we released a new BehaveX version that adds support for new report formatters, so I believe we are in shape to tackle this requirement. In the same way as the new Allure formatter is implemented, the approach would be to generate the Cucumber JSON output formatter based on the report.json that is generated as execution output. Do you believe this might be useful? 
We need to analyze if the screenshots and additional evidence attachments should also be provided with the Cucumber JSON formatter. 
Also, it is important to mention that when a formatter is set, the default HTML report is not generated, as the artifacts are managed in a different way by the formatter manager.
